src/routes/SelectRepos/actions.js

```diff
@@ -22,6 +22,11 @@ export const toggleRepoSelection = id => ({
   id,
 });
 
+export const resetSelectedRepos = id => ({
```

Comment:
this needs a test to be added to the actions.test file

Reply:
😕